### PR TITLE
Write unit tests for helpers

### PR DESCRIPTION
**Pull request by audy**
- wrote unit tests
- fixed one of the helper functions that was failing
- removed dummy fail test

### CODE REVIEW

**Comment by audy** [comment]
(part of #19)

File: src/helpers.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 // should this be configuration?
 var orcidRe = /(\d{4}-\d{4}-\d{4}-\d{4})@orcid\.org/;
+var Url = require('url')
```
Comment (acabunoc) [inline src/helpers.js]: haha, good catch

**Comment by acabunoc** [comment]
r+ for me. Good to merge.

**Comment by acabunoc** [comment]
Can you add text in README or CONTRIBUTING on how to run test?

**Comment by audy** [comment]
added to CONTRIBUTING

**Comment by acabunoc** [comment]
awesome! Merging

**Pull request merged**